src/FSharp.Data.Fred/JsonApi.fs

```diff
@@ -1,668 +1,356 @@
-        /// Order search by most recent real time start
-        | RealTimeStart 
-        /// Order search by most recent real time end
-        | RealTimeEnd
-        /// Order search by most recent updated series
-        | LastUpdated
-        /// Order search by most recent observation start date
-        | ObservationStart
-        /// Order search by most recent observation end date 
-        | ObservationEnd
-        /// Order search by most popular
-        | Popularity
-        /// Order search by most popular groups
-        | GroupPopularity
+[<RequireQualifiedAccessAttribute>]
+type SearchOrder = 
+    | SearchRank
+    | SeriesIdOrder
+    | Title
+    | Units
+    | Frequency
+    | SeasonalAdjustment 
+    | RealTimeStart 
+    | RealTimeEnd
+    | LastUpdated
+    | ObservationStart
+    | ObservationEnd
+    | Popularity
+    | GroupPopularity
 
-    ///   <summary>
-    ///    Sort results is ascending or descending observation date order.
-    ///   </summary>
-    ///   <category index="3">Unions</category>
-    [<RequireQualifiedAccessAttribute>]
-    type SortOrder =
-        /// Sort by ascending date order
-        | Ascending
-        /// Sort by descending date order
-        | Descending
+[<RequireQualifiedAccessAttribute>]
+type SortOrder =
+    | Ascending
+    | Descending
 
-    ///   <summary>
-    ///   A key that indicates a data value transformation.
-    ///   </summary>
-    ///   <category index="4">Unions</category>
-    [<RequireQualifiedAccessAttribute>]
-    type Units =
-        /// Observation(t), default.
-        | Levels
-        /// Observation(t) - Observation(t-1)
-        | Change
-        /// Observation(t) - Observation(t-#ObsPerYear)
-        | ChangeFromYearAgo
-        /// ((Observation(t)/Observation(t-1)) - 1) * 100
-        | PercentChange
-        /// ((Observation(t)/Observation(t-#ObsPerYear)) - 1) * 100
-        | PercentChangeFromYearAgo
-        /// (((Observation(t)/Observation(t-1)) ** (#ObsPerYear)) - 1) * 100
-        | CompoundedAnnualRateofChange
-        /// (ln(Observation(t)) - ln(Observation(t-1))) * 100
-        | ContinuouslyCompoundedRateofChange
-        /// ((ln(Observation(t)) - ln(Observation(t-1))) * 100) * #ObsPerYear
-        | ContinuouslyCompoundedAnnualRateofChange
-        /// ln(Observation(t))
-        | NaturalLog
+[<RequireQualifiedAccessAttribute>]
+type Units =
+    | Levels
+    | Change
+    | ChangeFromYearAgo
+    | PercentChange
+    | PercentChangeFromYearAgo
+    | CompoundedAnnualRateofChange
+    | ContinuouslyCompoundedRateofChange
+    | ContinuouslyCompoundedAnnualRateofChange
+    | NaturalLog
 
-    ///   <summary>
-    ///   The FRED frequency aggregation feature converts higher frequency data series into lower frequency 
-    ///   data series (e.g. converts a monthly data series into an annual data series). 
-    ///   In FRED, the highest frequency data is daily, and the lowest frequency data is annual.  
```

Comment:
The summary comment is good. You might be able to leave off the /// Daily frequency comments as they're mostly self explanatory.